scaladoc-testcases/src/tests/extensionParams.scala

```diff
@@ -53,11 +53,25 @@ extension (using String)(using Unit)(a: Animal)(using Int)(using Number)
   def f11(b: Any)(c: Any): Any
   = ???
 
+extension (a: Int)
+  def f13(): Any
+  = ???
+
+extension (using Unit)(a: Int)
+  def f14(): Any
+  = ???
+
+extension (a: Int)(using Unit)
+  def f15(): Any
+  = ???
```

Comment:
So looking at these, what do these tests check that the above ones won't check? For example the others seem to probably check the same thing right?

Reply:
So to list the different extensions tested in this file :
line 5 - 1 type/ 1 parameter
line 9, 13 & 17 -  1 type/ 1 parameter/ 1 implicit (The only difference is the function)
line 21 & 25 - 1 parameter/ 1 implicit
line 29 - 1 type bound/ 1 parameter/ 1 implicit
line 33, 37, 41, 45,  52 & 70- 1 type bound/ 2 implicit/ 1 parameter/ 2 implicit
So the three added extensions I have:
line 56 - 1 parameter
line 60 - 1 implicit/ 1 parameter
line 64 - 1 parameter/ 1 implicit

So as you can see most cases are tested but I notice the case of line 64 is already tested on line 21. Indeed it is a mistake on my part. So I think to remove this line (or change it by another case).